### PR TITLE
write api get all category

### PR DESCRIPTION
**Pull request by ghost**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Task ID: TI-95

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

**Review by mvn-haphan-dn — COMMENTED** [review]
LGTM

File: docs/api-references/category.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+### `GET` List Categories
+```
+/api/categories
+```
+Get list categories
+
+#### Query Param
+| Param | Type | Description |
+|---|---|---|
+| page | string | Paninate Books |
```
Comment (at-vietpham) [inline docs/api-references/category.md]: Thêm param cho phần limit

**Review by ghost — CHANGES_REQUESTED** [review]

File: docs/api-references/category.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+### `GET` List Categories
+```
+/api/categories
+```
+Get list categories
+
+#### Query Param
+| Param | Type | Description |
+|---|---|---|
+| page | string | Paninate Category |
```
Comment (at-vietpham) [inline docs/api-references/category.md]: Đây nên là Integer. Tương tự vơi limit ở dưới

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by at-vietpham** [comment]
LGTM :octocat:

**Pull request merged**